ddtrace/contrib/sqlite3/patch.py

```diff
@@ -75,3 +75,10 @@ def __init__(self, conn, pin=None, cursor_cls=None):
     def execute(self, *args, **kwargs):
         # sqlite has a few extra sugar functions
         return self.cursor().execute(*args, **kwargs)
+
+    def backup(self, target, *args, **kwargs):
```

Comment:
This is only available in Python 3.7+ we should only define this method if it exists in the base class.